packages/squiggle-lang/src/library/registry/frTypes.ts

```diff
@@ -358,18 +343,15 @@ export const frDictWithArbitraryKeys = <T>(
   };
 };
 
-export const frAny: FRType<Value> = {
-  unpack: (v) => v,
-  pack: (v) => v,
-  getName: () => "any",
-  transparent: true,
-};
-
-export const frGeneric = (index: string): FRType<Value> => ({
```

Comment:
The new syntax is quite verbose, I'd suggest keeping `frGeneric` as a shortcut, `export const frGeneric = (genericName: string) => frAny({ genericName })`.

Reply:
I don't feel like changing them all back now, that would take some time - and this way is more flexible, as we could more easily add ``keepBoxes`` where we want.

How about we keep, but add ``frGeneric``, and in the future, we can both choose to use that instead, and/or refactor as desired?

Reply:
Sure.